### DISCUSSION TITLE
Best way to put a drop case to the first letter

### DISCUSSION THREAD

**Discussion by kulbhushanchand** [mmistakes/minimal-mistakes#3454] [Q&A] [answered]
Hello everybody,

I tried adding the drop case to the first letter of the first paragraph on the blog post alone.
For this I try isolating the first paragraph as `.page__content > p:first-of-type:first-letter`, which works but also on all other pages. (Please see attached picture)
My intention is to put this change on the blog post only.

**What is the best way I can manage to select the first paragraph of the blog post only?**
**or any other way to do the same?**

Thanks

![image](https://user-images.githubusercontent.com/17742733/154266538-331eeff0-e5c3-49d9-a072-20353827f124.png)

**Comment by iBug** [comment, marked as answer]
Restrict your CSS selector to another HTML class, then [add `classes: example-class` to your posts](https://mmistakes.github.io/minimal-mistakes/docs/layouts/#wide-page) where you want to enable this feature, like this:

```scss
.example-class .page__content > p:first-of-type:first-letter {
  // Your CSS here
}
```

```yaml
---
title: Blah
classes: example-class
---

Post content
```

[This is what I'm doing][1] when I want to add page-specific CSS.

  [1]: https://github.com/iBug/iBug-source/blob/master/_pages/index.md?plain=1#L5

**Reply by kulbhushanchand** [reply]
Thanks @iBug, this looks like a smart (and probably the only) way instead of changing the layout files.
However, it also seems to be unnecessary repetition **if** I want every post under `_posts` folder to have this change.
- Does jekyll or minimal-mistakes provides any other way to target everything under the `_posts` folder?
- Does minimal-mistakes provide any specific css class for the files under the `_posts` folder only?

**Reply by iBug** [reply]
If you want to do something to all posts, use [Front Matter Defaults](https://jekyllrb.com/docs/configuration/front-matter-defaults/) to apply settings to all posts. This is stock Jekyll settings and should work decently.

**Reply by mmistakes** [reply]
This is where Jekyll's front matter defaults come into play.
https://jekyllrb.com/docs/step-by-step/09-collections/#front-matter-defaults

Example:

```yaml
# _config.yml
defaults:
  # _posts
  - scope:
      path: ""
      type: posts
    values:
      layout: single
     classes: "your-unqiue-class"
      ...

**Reply by kulbhushanchand** [reply]
Thanks a ton 🙏 @ibug and @mmistakes 
Front matter defaults is the one I was looking for.
Appreciate your kind support for this awesome theme...
